content/articles/how-to-create-a-rest-api-using-spring-boot-maven-and-mysql/index.md

```diff
@@ -0,0 +1,810 @@
+* MySQL Driver: It is a dependency that connects the MySQL database to a spring boot application.
+
+#### 1.2 Project Initialization and Setup Using [**Spring Initializer***](https://start.spring.io/)
+
+Alternatively, If you are not using IntelliJ you can use the [***Spring Initializer***](https://start.spring.io/) to set up your project. Enter the Maven project properties of the project including the dependencies as we did above and click on the generate button. This will generate a zip file and download it for you. Unzip it and open in your favorite IDEA and sync the dependencies with Maven.
+
+ ![spring-initializer-image](/engineering-education/how-to-create-a-rest-api-using-spring-boot-maven-and-mysql/spring-initializer.png)
+
+ ### 2. Maven Dependency Management
+The pom.xml file houses the dependencies, Maven plugins in our project.
+
+The dependency section simply contains the dependecies we added to our project namely  MySQL Driver, Spring Data JPA and SpringWeb. 
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>2.5.2</version>
+        <relativePath/> <!-- lookup parent from repository -->
+    </parent>
+    <groupId>com.lekwacious</groupId>
+    <artifactId>employee_app</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+    <name>employee_app</name>
+    <description>Demo project for Spring Boot</description>
+    <properties>
+        <java.version>11</java.version>
+    </properties>
+    <dependencies>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-data-jpa</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-web</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>mysql</groupId>
+            <artifactId>mysql-connector-java</artifactId>
+            <scope>runtime</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-test</artifactId>
+            <scope>test</scope>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.springframework.boot</groupId>
+                <artifactId>spring-boot-maven-plugin</artifactId>
+            </plugin>
+        </plugins>
+    </build>
+
+</project>
+```
+
+### 3. Configuring  MySQL, JPA, Hibernate
+In the resources folder within the src/main folder, open application.properties file and write the below properties.
```

Comment:
Always highlight keywords/variables/directory/filename using ` `` `

Here, highlight `src/main`

Reply:
Do this everywhere